CMakeLists.txt

```
@@ -107,8 +53,8 @@ find_package(catkin REQUIRED COMPONENTS
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES rt_assignment_2
+ INCLUDE_DIRS include
+ LIBRARIES rt_assignment_2
```

```suggestion
 CATKIN_DEPENDS message_runtime
```
This should not be library. You only need include and add `CATKIN_DEPENDS`. Additionally you need `message_runtime`